app/helpers/application_helper.rb

```diff
@@ -409,6 +409,10 @@ def edit_inline(object, property, options = {})
     end
   end
 
+  def foreman_doc_url(text = "")
```

Comment:
I'll change the name np, but about unifying I think it would be better to have both of them, one is for a button and one is for url, maybe ill just change the name of this method to documentation_url will that do?